database/migrations/2020_12_29_135542_add_route_to_archived_flights_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddRouteToArchivedFlightsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('archived_flights', function (Blueprint $table) {
+            $table->json('route')->nullable();
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('archived_flights', function (Blueprint $table) {
+            //
```

Comment:
Include something in the down method, for example `$table->dropColumn('route');`

Reply:
Resolved in 89edd33fbb72c544ea4e78e852b76e1f0427a456